Source/WebCore/css/FontSelectionValueInlines.h

```diff
@@ -120,8 +120,8 @@ inline std::optional<CSSValueID> fontStyleKeyword(std::optional<FontSelectionVal
 {
     if (!style || style.value() == normalItalicValue())
         return CSSValueNormal;
-    if (style.value() == italicValue())
-        return axis == FontStyleAxis::ital ? CSSValueItalic : CSSValueOblique;
+    if (style.value() == italicValue() && axis == FontStyleAxis::ital)
+        return CSSValueItalic;
```

Comment:
After the conversation in https://github.com/w3c/csswg-drafts/issues/8291, I think we may want to hold off this change for now. Sorry about that, the previous revision looked good.

Reply:
Pushed old build and it was already passing across all. I think we can merge it now. Let me know your thoughts before I add 'merge-queue'. Thanks!